tls/_common/prf.py

```diff
@@ -0,0 +1,69 @@
+# This file is dual licensed under the terms of the Apache License, Version
```

Comment:
It *looks* like if you move the docstring string up past the imports and just after the comment, it'll show up in sphinx.  Let me know if that helps!